app/build.gradle

```diff
@@ -18,6 +18,7 @@ android {
         versionName "1.0"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        buildConfigField "String", "APP_ID", "\"ff9d1c20c2cd49638fe55fcc5de49940\""
```

Comment:
let see if we can put this key in the local properties and no hardcoded here

Reply:
done